_config.yml

```diff
@@ -1,6 +1,6 @@
 title: DLF Metadata Assessment Working Group
 
-# baseurl: /Sandbox
+baseurl: /Sandbox
```

Comment:
Does this need to remain commented out?

Reply:
Yes, you are right. I've just edited the config.yml file and commented it out.